plugins/vbe/meson.build

```diff
@@ -0,0 +1,63 @@
+if get_option('plugin_vbe').enabled() or \
+   (get_option('plugin_vbe').auto() and libfdt.found())
```

Comment:
don't you also need a `libz.found()` for the second clause?

Reply:
Yes, done